docs/modules/setup/pages/diagram-options.adoc

```diff
@@ -12,13 +12,15 @@ Applies to `blockdiag`, `seqdiag`, `actdiag`, `nwdiag`, `packetdiag` and `rackdi
 
 |antialias
 |_boolean_ +
-`true`, *`false`*
-|Pass diagram image to anti-alias filter
+`true`, `false`
+|Apply anti-alias filter to diagram image +
+Option takes effect with both `true` and `false`
```

Comment:
Maybe we should say that the type is `_flag_` and that the option takes effect if the flag is present/specify/set. The value is actually ignored.

What do you think?

Reply:
> Maybe we should say that the type is `_flag_` and that the option takes effect if the flag is present/specify/set. The value is actually ignored.
> 
> What do you think?

Great idea. I reworked the PR, looks more natural to me now.